dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1842,7 +1857,7 @@ def is_int(x: str) -> bool:
             return a == b
 
     @staticmethod
-    def np_type_to_type(val: Any) -> Union[int, float]:
+    def np_type_to_type(val: Any) -> Union[int, float, Any]:
```

Comment:
if this can be any, we technically don't need union, right?

Reply:
However, will this error then bc we are using Any?

Reply:
I guess the Union was redundant. Changing to Any actually doesn't cause issues with our mypy flag `warn_return_any = True` because that only complains when we return Any and the function return type is not Any.

Reply:
nice